options/rtdl/generic/main.cpp

```diff
@@ -20,6 +20,16 @@
 #include <dlfcn.h>
 #endif
 
+#if defined(__i386__)
```

Comment:
Why is this necessary? Stack checking is disabled in rtdl.

Reply:
IIRC this was implemented by @FedorLap2006 so I can't speak to that.

Reply:
> Why is this necessary? Stack checking is disabled in rtdl.

It seems it's not used anywhere else. So I'm not sure why I put it there in the first place.